imjoy/util/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""Provide utilities that should not be aware of ImJoy engine."""
+
+
+class Registry(dict):
+    """Registry of items."""
+
+    # https://github.com/home-assistant/home-assistant/blob/
+    # 2a9fd9ae269e8929084e53ab12901e96aec93e7d/homeassistant/util/decorator.py
+    def register(self, name):
+        """Return decorator to register item with a specific name."""
+
+        def decorator(func):
+            """Register decorated function."""
+            self[name] = func
+            return func
+
+        return decorator
+
+
+def make_coro(func):
+    """Wrap a normal function with a coroutine."""
+
+    async def wrapper(*args, **kwargs):
```

Comment:
I wonder if this will blow up on Python 2 when we import this module?

Reply:
How do you test Python 2 plugins?

Reply:
add `"env": "conda create -n test-py2 python=2.7.16"` into `<config>` and you are ready to go.

Reply:
I've separated the Python 3 utils now.